armory/utils/metrics.py

```diff
@@ -174,6 +174,39 @@ def snr_spectrogram_db(x, x_adv):
     return [float(i) for i in 10 * np.log10(snr_spectrogram(x, x_adv))]
 
 
+def _image_circle_patch_area(x_i, x_adv_i):
+    if x_i.shape != x_adv_i.shape:
+        raise ValueError(f"x_i.shape {x_i.shape} != x_adv_i.shape {x_adv_i.shape}")
+    img_shape = x_i.shape
```

Comment:
Should return 0 if `x_i == x_adv_i`. Currently, it raises ValueError.

Reply:
addressed in ead1986